modules/ztp-nbde-deleting-old-tang-server-keys.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-nbde-implementation-guide.adoc
+
+[id="ztp-nbde-deleting-old-tang-server-keys_{context}"]
+= Deleting old Tang server keys
+
+.Prerequisites
+
+* A root shell on the Linux machine running the Tang server.
+
+.Procedure
+
+. Locate and access the directory where the Tang server key is stored. This is usually the `/var/db/tang` directory:
++
+[source,terminal]
+----
+# cd /var/db/tang/
+----
++
+[source,terminal]
+----
+# ls -A1
+----
++
+.Example output
+[source,terminal]
+----
+.36AHjNH3NZDSnlONLz1-V4ie6t8.jwk
+.gJZiNPMLRBnyo_ZKfK4_5SrnHYo.jwk
+Bp8XjITceWSN_7XFfW7WfJDTomE.jwk
+WOjQYkyK7DxY_T5pMncMO5w0f6E.jwk
+----
+
+. Delete the old keys:
++
+[source,terminal]
+----
+# rm .*.jwk
+----
++
+[source,terminal]
+----
+# ls -A1
+----
++
+.Example output
+[source,terminal]
+----
+Bp8XjITceWSN_7XFfW7WfJDTomE.jwk
+WOjQYkyK7DxY_T5pMncMO5w0f6E.jwk
+----
+
+.Verification
+
+At this point, the server still advertises the new keys, but an attempt to decrypt based on the old key will fail:
+
+[source,terminal]
+----
+# tang-show-keys 7500
+WOjQYkyK7DxY_T5pMncMO5w0f6E
```

Comment:
This line is output, not command; should it be in its own box?

Reply:
Put it in a new codeblock.